### DISCUSSION TITLE
Looking for your Feedback! Get $15 gift card for your time.

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#851] [Announcements]
Hi Community,

We are trying to learn more about how you use Leantime and how it has impacted your work. 
This will help us build new features and create a roadmap. 

For that we're asking for 30min phone calls to dive deeper into your use case. In return we'll send you a $15 Amazon gift card and the chance to be featured on our website. 

Please sign up here if you are interested: https://calendly.com/marcelfolaron/leantime-user-feedback?month=2022-08

Looking forward to chatting!

**Comment by broskees** [comment]
@marcelfolaron - link is still for august - unless this is finished.
